rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java

```diff
@@ -45,6 +46,12 @@
         List<BtcECKey> genesisFederationPublicKeys = Lists.newArrayList(
                 federator0PublicKey, federator1PublicKey, federator2PublicKey
         );
+
+        // IMPORTANT: Both BTC and RSK keys are the same.
+        // Change upon implementation of the <INSERT FORK NAME HERE> fork.
+        List<FederationMember> federationMembers = genesisFederationPublicKeys.stream().map(pk -> new FederationMember(
```

Comment:
Not important at all, but given the `genesisFederationPublicKeys` list is not used elsewhere it could be rewritten to
```java
List<FederationMember> federationMembers = Stream.of(federator0PublicKey, federator1PublicKey, federator2PublicKey)
        .map(pk -> new FederationMember(pk, ECKey.fromPublicOnly(pk.getPubKey())))
        .collect(Collectors.toList());
```

Reply:
This will very likely disappear anyway when we implement different keys for each `FederationMember`. Leaving as is.